app/assets/javascripts/select-kit/components/user-tracking-dropdown.js.es6

```diff
@@ -0,0 +1,76 @@
+import DropdownSelectBox from "select-kit/components/dropdown-select-box";
+
+export default DropdownSelectBox.extend({
+  pluginApiIdentifiers: [""],
+  classNames: ["user-tracking", "user-tracking-dropdown"],
+  nameProperty: "label",
+  allowInitialValueMutation: false,
+
+  computeHeaderContent() {
+    let content = this._super(...arguments);
+    if(this.get("user.ignored")) {
+      this.set("headerIcon", "eye-slash");
+      content.name = `${I18n.t("user.user_tracking_ignore_option")}`;
+    } else if(this.get("user.muted")) {
+      this.set("headerIcon", "times-circle");
+      content.name = `${I18n.t("user.user_tracking_mute_option")}`;
+    } else {
+      this.set("headerIcon", "user");
+      content.name = `${I18n.t("user.user_tracking_normal_option")}`;
+    }
+    return content;
+  },
+
+  computeContent() {
+    const content = [];
+
+    content.push({
```

Comment:
Rater than creating an array and calling `push`, you could return the array as a literal:

```
return [
  firstThing,
  secondThing,
  etc
]